shortfin/python/shortfin_apps/sd/server.py

```diff
@@ -135,7 +136,7 @@ def get_configs(args):
         f"--model={modelname}",
         f"--topology={topology_inp}",
     ]
-    outs = subprocess.check_output(cfg_builder_args).decode()
+    outs = subprocess.check_output(cfg_builder_args, stderr=subprocess.DEVNULL).decode()
```

Comment:
This makes debugging impossible... We really need to finish the build API and get this stuff more for free